spring-kafka/src/main/java/org/springframework/kafka/config/AbstractKafkaListenerContainerFactory.java

```diff
@@ -402,7 +403,8 @@ private void configureEndpoint(AbstractKafkaListenerEndpoint<K, V> aklEndpoint)
 	protected void initializeContainer(C instance, KafkaListenerEndpoint endpoint) {
 		ContainerProperties properties = instance.getContainerProperties();
 		BeanUtils.copyProperties(this.containerProperties, properties, "topics", "topicPartitions", "topicPattern",
-				"messageListener", "ackCount", "ackTime", "subBatchPerPartition");
+				"messageListener", "ackCount", "ackTime", "subBatchPerPartition", "kafkaConsumerProperties");
+		properties.setKafkaConsumerProperties(new Properties(this.containerProperties.getKafkaConsumerProperties()));
```

Comment:
I comment out this line of code and no one test fails, even your new one.
What am I missing in the logic?
Thanks

Reply:
This code is not necessary; the constructor initializes the field with a new `Properties`.

Reply:
OK. Merging your PR then, Gary.